src/jsMain/kotlin/io/beatmaps/user/alerts/main.kt

```diff
@@ -54,6 +54,7 @@ class AlertsPage : RComponent<AlertsPageProps, AlertsPageState>() {
 
             setState {
                 alertStats = data
+                hiddenAlerts = listOf()
```

Comment:
Literally can't tell the difference, there shouldn't be one

Reply:
When logging out, logging in again and going to your alerts page, the alerts don't disappear when you click the eye. Once you change the a filter or go to your read alerts, they do.

Reply:
Ah, would probably prefer this be handled on line 192
`hiddenAlerts = (hiddenAlerts ?: emptyListOf()).plus(id)`